### DISCUSSION TITLE
跪求Doris Manager部署文件

### DISCUSSION THREAD

**Discussion by 666chaoge666** [apache/doris#9019] [A - General / Q&A]
跪求官网中Doris Manager的JAR包，配置文件夹及启动停止脚本
或者源码也可

<img width="779" alt="image" src="https://user-images.githubusercontent.com/77181242/163209199-c5f795fe-a27e-4565-8c40-2ab61c7b9906.png">

**Comment by JNSimba** [comment]
https://github.com/apache/incubator-doris-manager
Compilation documentation can refer to：https://github.com/apache/incubator-doris-manager/blob/master/docs/Doris%20Manager%20Compiling%20%26%20Deploying.md
